kv/range_cache.go

```diff
@@ -78,7 +78,11 @@ type RangeDescriptorDB interface {
 	// returned. The first of these slices holds descriptors which contain
 	// the given key (possibly from intents), and the second being prefetched
 	// adjacent descriptors.
-	RangeLookup(roachpb.RKey, *roachpb.RangeDescriptor, bool, bool) ([]roachpb.RangeDescriptor, []roachpb.RangeDescriptor, *roachpb.Error)
+	// TODO(andrei): Should this return error instead of pErr?
```

Comment:
Yes, I think this should also be `error` to match the rest of the `RangeCache` interface.  - there's no need to return a `pError` if `performRangeLookup` is just going to call `GoError` on it.